Makefile.js

```diff
@@ -708,6 +736,51 @@ target.checkLicenses = function() {
     });
 };
 
+target.checkGitCommit = function() {
+    var commitMsgs,
+        failed;
+
+    if (hasBranch("master")) {
```

Comment:
Yeah, I'd say it's fine to remind the single commit check in order to get the commit message format fix in.

It's also trivial for the bot to see if there are multiple commits, so I could add it there as well.